solr/contrib/ltr/src/test/org/apache/solr/ltr/ranking/TestModelQuery.java

```diff
@@ -229,14 +230,14 @@ public void testModelQuery() throws IOException, ModelException {
     for (int i = 0; i < 3; i++) {
       assertEquals(i, modelWeight.modelFeatureValuesNormalized[i], 0.0001);
     }
-    for (int i = 0; i < 10; i++) {
-      assertEquals(i, modelWeight.allFeatureValues[i], 0.0001);
-    }
-
-    for (int i = 0; i < 10; i++) {
-      assertEquals("f" + i, modelWeight.allFeatureNames[i]);
+  //  for (int i = 0; i < 10; i++) {
+  //    assertEquals(i, modelWeight.allFeatureValues[i], 0.0001);
```

Comment:
Should put these tests past to correctly assert things based on the 3 different combinations of all vs model features